src/repositories/cart.ts

```diff
@@ -54,15 +54,19 @@ const updateUserCart = async (userId: string, product: ProductData): Promise<Car
 
             return await activeCart.save()
                 .then(savedCart => fetchCartAndTotalPrice(savedCart))
-                .catch(err => null)
+                .catch(() => {
+                    return null;
+                })
         }
 
         if (productIsInCart) {
             activeCart.items[productIndex].count = activeCart.items[productIndex].count + product.count;
 
             return await activeCart.save()
                 .then(savedCart => fetchCartAndTotalPrice(savedCart))
-                .catch(err => null)
+                .catch(() => {
```

Comment:
please use `try catch`
And I see you dont handle error, at least add `logger` with error message, cause it will be hard to understand what went wrong with this call